src/tactic/ui/panel/hash_panel_wdg.py

```diff
@@ -162,11 +162,34 @@ def _get_predefined_url(cls, key, hash):
                 # put in a check to ensure this is a user
                 parts = link.split(".")
 
+                '''For usernames with at most one period
                 # you can only see your personal links
                 user = Environment.get_user_name()
                 if user == parts[0]:
                     personal = True
+                elif parts[1]:
```

Comment:
would that possibly cause an IndexError .. what if link has no . in it at all. This goes for parts[0] as well . Can u check that?
Actually why dont' we use the is_personal method below to handle all of these situation so it's more contained?